fdsc.user.js

```diff
@@ -1,8 +1,8 @@
 // ==UserScript==
 // @name        Flag Dialog Smokey Controls
 // @description Adds Smokey status of a post and feedback options to flag dialogs.
-// @author      ArtOfCode
-// @version     0.14.1
+// @author      ArtOfCode (see source for contributors)
+// @version     0.15.1
```

Comment:
Gonna leave the version number at 0.15.1 until it's been out in the wild for a while, just to make sure there are no bugs left in this version before we move to 1.0.0